NorthstarDLL/squirrel/squirrel.cpp

```diff
@@ -178,37 +179,41 @@ template <ScriptContext context> void SquirrelManager<context>::GenerateSquirrel
 	s->__sq_pushasset = __sq_pushasset;
 	s->__sq_pushvector = __sq_pushvector;
 	s->__sq_pushobject = __sq_pushobject;
-	s->__sq_getstring = __sq_getstring;
-	s->__sq_getthisentity = __sq_getthisentity;
-	s->__sq_getobject = __sq_getobject;
-
-	s->__sq_stackinfos = __sq_stackinfos;
 
+	s->__sq_getstring = __sq_getstring;
 	s->__sq_getinteger = __sq_getinteger;
 	s->__sq_getfloat = __sq_getfloat;
 	s->__sq_getbool = __sq_getbool;
 	s->__sq_get = __sq_get;
 	s->__sq_getasset = __sq_getasset;
 	s->__sq_getuserdata = __sq_getuserdata;
 	s->__sq_getvector = __sq_getvector;
+	s->__sq_getthisentity = __sq_getthisentity;
+	s->__sq_getobject = __sq_getobject;
+
+	s->__sq_stackinfos = __sq_stackinfos;
```

Comment:
Same question as above regarding re-ordering.

Reply:
to match the `SquirrelFunctions` order